.github/workflows/release.yaml

```diff
@@ -80,8 +131,6 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   release-gnu:
```

Comment:
If we support arm64, can we add a non-os-specific linux release for this as well? This container should make it possible: https://quay.io/repository/pypa/manylinux2014_aarch64

Reply:
Sure, I can try it out.